dev-requirements.txt

```diff
@@ -3,3 +3,5 @@ sphinx>=1.1
 colour>=0.0.5
 mock>=1.0
 pep8>=1.5.7
+google-api-python-client>=1.4.2
+python-dateutil>=2.4.2
```

Comment:
No pb, I have no right to critisize ^^ thanks for considering it :)

2016-02-14 11:21 GMT+01:00 enkore <notifications@github.com>:

> In dev-requirements.txt
> <https://github.com/enkore/i3pystatus/pull/317#discussion_r52840556>:
>
> > @@ -3,3 +3,5 @@ sphinx>=1.1
> >  colour>=0.0.5
> >  mock>=1.0
> >  pep8>=1.5.7
> > +google-api-python-client>=1.4.2
> > +python-dateutil>=2.4.2
>
> These two should go into MOCK_MODULES in docs/conf.py: the
> dev-requirements should only be used for packages needed to run the tests
> etc.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/enkore/i3pystatus/pull/317/files#r52840556>.
>